iso:std:iso:34000:ed-1:v1:en.xml

```diff
@@ -0,0 +1,1556 @@
+      <p>Specifically, the terminology presented in this document:</p>
+      <list list-type="dash">
+        <list-item>
+          <label>&#x2014;</label>
+          <p>serves as a sound basis in the understanding of date and time;</p>
+        </list-item>
+        <list-item>
+          <label>&#x2014;</label>
+          <p>guides new developments in the field by underpinning mutual understanding;</p>
+        </list-item>
+        <list-item>
+          <label>&#x2014;</label>
+          <p>serves as a quick and handy reference for those newly inaugurated to this field.</p>
+        </list-item>
+      </list>
+      <p>In this document, the decimal sign is a comma on the line, and each group of three digits are separated by a small space from the preceding digits, counting from the decimal sign, in accordance with the ISO/IEC Directives, Part 2.</p>
+      <p>However, Resolution 10 of the 22nd General Conference on Weights and Measures (Conf&#xE9;rence G&#xE9;n&#xE9;rale des Poids et Mesures, CGPM) in 2003 stated that:</p>
+      <p>&#x201C;The decimal marker shall be either a point on the line or a comma on the line.&#x201D;</p>
+      <p>And reaffirmed the following resolution from Resolution 7 of the 9th CGPM, 1948:</p>
+      <p>&#x201C;Numbers may be divided in groups of three in order to facilitate reading.&#x201D;</p>
+      <p>In practice, the choice between these alternatives depends on customary use in the language concerned. In the technical areas of date and time, it is customary for the decimal point always to be used, and that numbers are not grouped, for all languages.</p>
+    </sec>
+  </front>
+  <body>
+    <sec id="sec_1">
+      <label>1</label>
+      <title>1   Scope</title>
+      <p>This document defines terms related to date and time, from fundamental concepts to those of their usage and representation.</p>
+    </sec>
+    <sec id="sec_2">
+      <label>2</label>
+      <title>2   Normative references</title>
+      <p>There are no normative references in this document.</p>
+    </sec>
+    <sec id="sec_3">
+      <label>3</label>
+      <title>3   Terms related to general concepts</title>
+      <p>ISO and IEC maintain terminology databases for use in standardization at the following addresses:</p>
+      <list list-type="dash">
+        <list-item>
+          <label>&#x2014;</label>
+          <p>ISO Online browsing platform: available at</p>
+        </list-item>
+        <list-item>
+          <label>&#x2014;</label>
+          <p>IEC Electropedia: available at</p>
+        </list-item>
+      </list>
+      <term-sec id="sec_3.1">
+        <label>3.1</label>
+        <tbx:termEntry id="term_3.1">
+          <tbx:langSet xml:lang="en">
+            <tbx:tig>
+              <tbx:term>date</tbx:term>
+              <tbx:partOfSpeech value="noun"/>
+            </tbx:tig>
```

Comment:
How to generate `tbx:normativeAuthorization` in the `tbx:tig` section ?

Reply:
I've also just looked at [the documentation about this tag](https://www.niso-sts.org/TagLibrary/niso-sts-TL-1-0-html/element/tbx-normativeAuthorization.html) and the expected values.

Once again, the HTML doesn't contain the necessary information to know what value to set.

The same will apply to the following tags in this section:
- `termType`
- `grammaticalGender`

And surely others when I look at the documentation related to `tbx:`.

Reply:
When the term is not a `noun` (default), it will be indicated in the HTML.

A "deprecated" term is italicized, an "admitted" term is not bolded, a "preferred" term (default) is bolded.